admin-ui/src/component/reservations/requested/DenyDialog.tsx

```diff
@@ -87,6 +178,27 @@ const DialogContent = ({
     },
   });
 
+  const { notifyError, notifySuccess } = useNotification();
+  const { t } = useTranslation();
+
+  const [refundReservationMutation] = useMutation<Mutation>(
```

Comment:
Might be useful to define input/payload types here.

Reply:
Good call